### ISSUE TITLE
Latest update 

### ISSUE THREAD

**Issue by miker1117** [lubomyr/uae4arm#27]
I am quick check out of this latest update

Fixed, selection of the bottom drop down (a4000) and on the inputs no longer FC the apl

Broke:

Configs from last version cause FC, I believe this could be due to the fact that selecting JIT, causes a FC regardless, I checked the last version and it works fine. 
Still have to assign your buttons in the SDL, haven't been able to set in the app itself.. (Mouse buttons) 

I went ahead and put in my HDDS, with the latest scalos, mui, and and whdload .. AS OF A YEAR OR SO AGO
Everything seemed fine, except the whdload icon to run the game wasn't showing up, from past experience is a whdload issue, ill check into it on the last version.. 
That's it for now.

**Comment by miker1117** [comment]
Update:
OK, I this new version seems to have issues with the assigns, as mentioned the Whdload icons arent showing, they are on the last version with the exact same is hdd, click the game launch a d its unassigned tool, downloaded whdload, and and install gives the same, and I select the install tool and it still doesn't work.... 

Now I'm going to paste my Assigns, and as this could very well be a good issue on my end, I but it works fine on last version.... 

File:My-Assigns


Assign >NIL: C: Shared:S-System/C
Assign >NIL: S-System: Shared:S-System
Assign >NIL: Fonts: Shared:S-System/Fonts
Assign >NIL: DEVS: Shared:S-System/DEVS
Assign >NIL: Classes: Shared:S-System/Classes
Assign >NIL: Libs: Shared:S-System/Libs
Assign >NIL: Icons: Shared:S-System/Icons
Assign >NIL: L: Shared:S-System/L
Assign >NIL: Locale: Shared:S-System/Locale
Assign >NIL: MyFiles: Shared:S-System/MyFiles-P96
Assign >NIL: Programs: Shared:S-System/Programs
Assign >NIL: Tools: Shared:S-System/Tools
Assign >NIL: Utilities: Shared:S-System/Utilities
Assign >NIL: Preferences: Shared:S-System/Preferences
Assign >NIL: Rexxc: Shared:S-System/Rexxc
Assign >NIL: Classes: Shared:S-System/Classes
Assign >NIL: LIBS: S-System:Classes ADD
Assign >NIL: SystemS: Shared:S-System/SystemS
Assign >NIL: MultiViewNot: Shared:S-System/Programs/FinalWriter/FinalWriter
Assign >NIL: Work: Shared:S-System/Work
Assign >NIL: SysUtilities: Shared:SysUtilities
;BEGIN Genesis
assign AmiTCP: "Shared:S-System/Genesis"
;END Genesis

File:My-Paths

Path >NIL: RAM: C: S-SYSTEM:Utilities S-SYSTEM:Rexxc SYS:System S: SYS:Prefs SYS:WBStartup S-SYSTEM:Tools S-SYSTEM:Tools/Commodities

Basically all my installed os' use the same base files for all, but as I don't have 5 copies of the same install on all os'

Sounds confusing, but hope it helps..

**Comment by miker1117** [comment]
Update to the update
OK there is something wrong, as far as I can tell with the tools association, I decided to reinstall from scratch WB 3.10, put install 43 in C and went to install Picasso96, standard practice before mui and scalos, well it doesn't even show setup, but for the picasoinstall file, booting in the previous version, same exact HDD, it shows the setup with the proper icon, which kind of explains the previous issue with Whdload.. 

I don't know enough about the Amiga to continue trying to trouble shoot this, other then doing the installs on the previous version, which would defeat the purpose.. So I'm stuck..
